.github/workflows/docs.yml

```diff
@@ -174,17 +175,6 @@ jobs:
           cp -r doc/examples/ ./pyvista-examples/
           ls -l ./pyvista-examples/
 
-      - name: Publish notebooks on main
-        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
-        uses: peaceiris/actions-gh-pages@v3
-        with:
-          external_repository: pyvista/pyvista-examples
-          personal_token: ${{ secrets.PYVISTA_BOT_TOKEN }}
-          publish_dir: pyvista-examples
-          publish_branch: develop
-          exclude_assets: ''
-          force_orphan: true
```

Comment:
These are pushing to the same repo as the release and thus overwriting the examples from the release. The docker image there uses the `latest` tag so it's very likely that the examples will mismatch the PyVista version there